src/nvim/memory.c

```diff
@@ -496,7 +496,7 @@ void free_all_mem(void)
   block_autocmds();
 
   /* Close all tabs and windows.  Reset 'equalalways' to avoid redraws. */
-  p_ea = FALSE;
+  p_ea = false;
```

Comment:
For a global variable, I think it's better changing all occurrences at once instead of just those that happen to be in a single file, whenever reasonable. As `p_ea` and `p_acd` are not so heavily used, I would suggest fixing the rest of them (incl the definition type) in this PR.